src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -419,164 +367,108 @@ def list_connectedk8s(cmd, client, resource_group_name=None):
+            if release['name'] == 'azure-arc':
+                return release['namespace']
+    return None
 
-def create_aad_application(cmd, display_name, homepage=None, identifier_uris=None,  # pylint: disable=too-many-locals
-                       available_to_other_tenants=False, password=None, reply_urls=None,
-                       key_value=None, key_type=None, key_usage=None, start_date=None, end_date=None,
-                       oauth2_allow_implicit_flow=None, required_resource_accesses=None, native_app=None,
-                       credential_description=None, app_roles=None):
-    graph_client = _graph_client_factory(cmd.cli_ctx)
-    password_creds = [PasswordCredential(start_date=start_date,
-                                         end_date=end_date, key_id=str(uuid.uuid4()),
-                                         value=password,
-                                         custom_key_identifier=None)]
-    app_create_param = ApplicationCreateParameters(available_to_other_tenants=False,
-                                                   display_name=display_name,
-                                                   identifier_uris=identifier_uris,
-                                                   homepage='https://' + display_name,
-                                                   reply_urls=None,
-                                                   key_credentials=None,
-                                                   password_credentials=password_creds,
-                                                   oauth2_allow_implicit_flow=None,
-                                                   required_resource_access=None,
-                                                   app_roles=None)
-    try:
-        result = graph_client.applications.create(app_create_param)
-    except GraphErrorException as ex:
-        if 'insufficient privileges' in str(ex).lower():
-            link = 'https://docs.microsoft.com/azure/azure-resource-manager/resource-group-create-service-principal-portal'  # pylint: disable=line-too-long
-            raise CLIError("Directory permission is needed for the current user to register the application. "
-                           "For how to configure, please refer '{}'. Original error: {}".format(link, ex))
-        raise
-    return result
-
-
-def _create_service_principal(cli_ctx, identifier, resolve_app=True):
-    client = _graph_client_factory(cli_ctx)
-    app_id = identifier
-    if resolve_app:
-        if _is_guid(identifier):
-            result = list(client.applications.list(filter="appId eq '{}'".format(identifier)))
-        else:
-            result = list(client.applications.list(
-                filter="identifierUris/any(s:s eq '{}')".format(identifier)))
 
+def delete_cc_resource(client, resource_group_name, cluster_name):
+    try:
+        client.delete(resource_group_name, cluster_name)
+    except CloudError as ex:
+        raise CLIError(ex)
```

Comment:
Question: What's printed in such cases? Entire verbose content of error?  Also would this be printed even without user specifying --debug?